jenkins/spark-premerge-build.sh

```diff
@@ -19,12 +19,16 @@ set -ex
 
 nvidia-smi
 
-function on_exit {
-    echo '### END OF TEST LOG ###'
-}
-trap on_exit EXIT
+if [[ ! -z "$RUN_UT_ONLY" ]]; then
```

Comment:
I think we could be better to add the full branches here like,
1. VERIFY_ONLY || ALL
2. UT_ONLY || ALL

So this is more people to modify and keep it simplicity if need to bring it back to sequential stages

Reply:
I'm trying to minimize the change. If need cover above more cases, I would suggest make this script parameterized, such as `xxx-build.sh [TYPE]`, e.g, `xxx-build.sh UT`, `xxx-build.sh IT` and `all` by default

Reply:
> To avoid confusing with yes/no, true/false (and case sensitive), usually would test whether it's empty or not

sure, but this this one. I think to test if empty maybe not enough, `RUN_UT_ONLY=yes` people would obviously considering to disable that they should set it as RUN_UT_ONLY=no, but this won't work

Reply:
talked offline. If adding arg, please make sure its `VERIFY` instead of `IT` as above 